qiita_db/test/test_processing_job.py

```diff
@@ -282,6 +282,75 @@ def test_submit_error(self):
                 qdb.exceptions.QiitaDBOperationNotPermittedError):
             job.submit()
 
+    def test_complete_multiple_outputs(self):
+        # This test performs the test of multiple functions at the same
+        # time. "release", "release_validators" and
+        # "_set_validator_jobs" are tested here for correct execution.
+        # Those functions are designed to work together, so it becomes
+        # really hard to test each of the funrcions individually for
+        # successfull execution.
+        # We need to create a new command with multiple outputs, since
+        # in the test DB there is no command with such caracteristics
+        cmd = qdb.software.Command.create(
+            qdb.software.Software(1),
+            "TestCommand", "Test command",
+            {'input': ['artifact:["Demultiplexed"]', None]},
+            {'out1': 'BIOM', 'out2': 'BIOM'})
+        job = qdb.processing_job.ProcessingJob.create(
+            qdb.user.User('test@foo.bar'),
+            qdb.software.Parameters.load(
+                cmd,
+                values_dict={"input": 1}))
+        job._set_status("running")
+
+        fd, fp1 = mkstemp(suffix="_table.biom")
+        self._clean_up_files.append(fp1)
+        close(fd)
+        with open(fp1, 'w') as f:
+            f.write('\n')
+
+        fd, fp2 = mkstemp(suffix="_table.biom")
+        self._clean_up_files.append(fp2)
+        close(fd)
+        with open(fp2, 'w') as f:
+            f.write('\n')
+
```

Comment:
As a novice Qiita dev, a little more inline documentation on what's going on with the tests would be helpful, since the Qiita artifact and job creation concept is a little difficult to grasp. Not blocking for this PR but in general would be helpful to keep in mind...